src/Microsoft.AspNetCore.Server.Kestrel/Internal/KestrelServerConfigureOptions.cs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.AspNetCore.Certificates.Configuration;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal
+{
+    public class KestrelServerConfigureOptions : IConfigureOptions<KestrelServerOptions>
```

Comment:
You can't combine them as one is a true default that you always want to apply, while the config binding one is a 'squishy' default, that we only want to apply by default via WebHostBuilder